dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3DeleteFilesTaskTest.java

```diff
@@ -45,31 +60,66 @@
+    private static final String TEST_ENV_NAME = "DeleteFilesTaskTest";
+    private static final String DELETE_FILES_TASK_TEST_BUCKET_NAME = "DeleteFilesTaskTest_Bucket";
+    private static final BuildInfoServiceImpl buildInfoService = new BuildInfoServiceImpl();
+    private static TempStorageIds envStorageIds;
+    private static UUID envDataPolicyId;
 
     @Before
     public void setUp() {
         new JFXPanel();
         Platform.runLater(() -> {
-            final SavedSession savedSession = new SavedSession(SessionConstants.SESSION_NAME,
-                    SessionConstants.SESSION_PATH, SessionConstants.PORT_NO, null,
-                    new SavedCredentials(SessionConstants.ACCESS_ID, SessionConstants.SECRET_KEY),
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
                     false);
             session = new CreateConnectionTask().createConnection(
-                    SessionModelService.setSessionModel(savedSession, false));
+                    SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            try {
+                envDataPolicyId = IntegrationHelpers.setupDataPolicy(TEST_ENV_NAME, false, ChecksumType.Type.MD5, client);
+                envStorageIds = IntegrationHelpers.setup(TEST_ENV_NAME, envDataPolicyId, client);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
         });
     }
 
+    @After
+    public void teardown() throws IOException {
+        client.deleteBucketSpectraS3(new DeleteBucketSpectraS3Request(DELETE_FILES_TASK_TEST_BUCKET_NAME).withForce(true));
+        IntegrationHelpers.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
     @Test
-    public void deleteFiles() throws Exception {
+    public void deleteFiles() throws InterruptedException {
         final CountDownLatch latch = new CountDownLatch(1);
         Platform.runLater(() -> {
             try {
-                final ImmutableList<String> buckets = new ImmutableList.Builder<String>()
-                        .add(SessionConstants.DS3_PANEL_SERVICE_TEST_BUCKET_NAME)
-                        .build();
+                HELPERS.ensureBucketExists(DELETE_FILES_TASK_TEST_BUCKET_NAME, envDataPolicyId);
+                Path path = null;
+                try {
+                    path = ResourceUtils.loadFileResource("files/");
+                } catch (URISyntaxException | FileNotFoundException e) {
+                    e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.